analysis/watason_personality.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
```

Comment:
I'll approve it, and if you don't want to change it right now, i can do it later, but the file is called `watason_personality.js` as a heads up.

Reply:
Whoops good catch I'll update that before merging